src/components/canrawview/canrawview_p.h

```diff
@@ -97,7 +108,11 @@ private slots:
     void sort(const int currentIndex, QList<int> &clickedIndexes)
     {
         clickedIndexes.removeFirst();
-        clickedIndexes.append(currentIndex);
+
+        if (currentIndex == 2)
```

Comment:
Get rid of magic numbers. You can use global constants in unnamed namespace instead.